.github/workflows/build.yaml

```
@@ -205,7 +206,13 @@ jobs:
         build/src/website/wl_map_object_info temp_web || [ $? -eq 2 ]
         build/src/website/wl_map_info data/maps/Archipelago_Sea.wmf || [ $? -eq 2 ]
     - name: Testsuite
-      run: ./regression_test.py -b build/src/widelands
+      run: |
+        TIMEOUT=""
+        if [ "$CONF" = "Debug" ]
```

Is there a special reason, why you do not use matrix.config here directly?
```suggestion
        if [ "${{ matrix.config }}" = "Debug" ]
```

---

Do you mean other than not being familiar with it? :smiley:

Thanks!